src/app/modules/practicas-academicas/detalle-practica-academica/detalle-practica-academica.component.ts

```diff
@@ -95,8 +95,8 @@ export class DetallePracticaAcademicaComponent {
           .get("practicas-academicas/" + id)
           .subscribe((practica) => {
             const r = <any>practica;
-            if (practica !== null && r.success !== false) {
-              if (r.status === 200 && practica["data"] !== null) {
+            if (practica !== null && r.Success !== false) {
+              if (r.Status === 200 && practica["data"] !== null) {
```

Comment:
Este microcliente tiene una forma rara de usar la respuesta de data, fíjese que practica es el response.. lo mandan a una variable nueva para consultar el status pero data sigue estando en practica["Data"]
```suggestion
              if (r.Status === 200 && practica["Data"] !== null) {
```